src/Workspaces/Core/MSBuild/MSBuild/ProjectMap.cs

```diff
@@ -11,11 +11,14 @@
 
 namespace Microsoft.CodeAnalysis.MSBuild
 {
+
+#pragma warning disable CS0419 // Ambiguous reference in cref attribute
```

Comment:
What is the compiler emitting for the cref then -- is it just dropping it or doing something else funky?